pkg/autodiscovery/autoconfig.go

```diff
@@ -219,59 +233,15 @@ func (ac *AutoConfig) HasRunOnce() bool {
 }
 
 // GetAllConfigs queries all the providers and returns all the integration
-// configurations found, resolving the ones it can
+// configurations found.
 func (ac *AutoConfig) GetAllConfigs() []integration.Config {
-	return ac.getAllConfigs().schedule
-}
-
-// getAllConfigs queries all the providers and returns all the integration
-// configurations found, resolving the ones it can, and returns a configChanges to
-// schedule all of them.
-func (ac *AutoConfig) getAllConfigs() configChanges {
-	changes := configChanges{}
+	configs := []integration.Config{}
```

Comment:
nit: if the slice can be nil, you can replace it by

```suggestion
	var configs []integration.Config
```